pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -22,6 +22,7 @@ stdenv.mkDerivation {
 
   passthru = {
     BL31_RK3568 = "${rkbin}/bin/rk35/rk3568_bl31_v1.44.elf";
+    BL31_RK3588 = "${rkbin}/bin/rk35/rk3588_bl31_v1.45.elf";
```

Comment:
The BL31 for RK3588 is built by `armTrustedFirmwareRK3588`, so this isn't really necessary.
https://github.com/NixOS/nixpkgs/blob/1db686b48bf47119f5782a62efd9b905d911e17c/pkgs/misc/uboot/default.nix#L537

Reply:
Ah I missed that! Sounds good, I'll close